src/interface.js

```diff
@@ -14,9 +14,6 @@ module.exports = function(Target) {
 
   // Format the query as sql, prepping bindings as necessary.
   Target.prototype._formatQuery = function(sql, bindings, tz) {
-    if (this.client && this.client.prepBindings) {
-      bindings = this.client.prepBindings(bindings, tz);
-    }
```

Comment:
@elhigu I'm unsure on this one. It was added in [5b9336a28453fe62e4b412f1c021512a65ae6027](https://github.com/tgriesser/knex/commit/5b9336a28453fe62e4b412f1c021512a65ae6027) 2 years ago, but I cant find `toQuery()` being called with an argument. I also don't know if this has any significance to the exposed `toQuery()` function seeing as this `tz` argument let alone `toQuery` is not documented at all. Only `toSQL` and `toString` are in the docs. I'll add it as an argument to toSQL for now, like you said.

@tgriesser Can you weigh in here?